cloudformation.yaml

```diff
@@ -10,11 +10,7 @@
 # and limitations under the License.
 
 AWSTemplateFormatVersion: 2010-09-09
-Parameters:
-  CodeKey:
-    Type: String
-    Description: >-
-      The key that the lambda function code has been uploaded to
+%PARAMETERS%
```

Comment:
I'm not familiar with this syntax for yaml and can't seem to find reference to this syntax online. What is this doing?

Reply:
Hmm I think that was already present from the template I built off from but I think it might have something to do with pulling parameters from what event engine creates? I am not 100% sure. @praddc  do you have any idea?

Reply:
Weird, I haven't seen that before either, but I'm going to guess it's just a placeholder since there are no Parameters needed by this CFn. You could also just omit this without issue.